README.md

```diff
@@ -98,6 +98,22 @@ const channel = await getOrCreateChannel('myChannel')
 channel.broadcast({ ... })
 ```
 
+To prevent the API from being used before it is ready, a global `fdc3Ready` event is provided:
+
+```js
+function fdc3Stuff() {
+  // Make some fdc3 API calls here
+}
+
+if (window.fdc3) {
+  fdc3Stuff();
+} else {
+  window.addEventListener("fdc3Ready", fdc3Stuff);
+}
+```
+
+[comment]: # (insert instructions for fdc3Ready event use via ES6 syntax here)
+
```

Comment:
@kriswest I think we should leave this out of the README for now. It is not meant to be exhaustive, the README just has some very basic examples. I think the right place to properly document it is through the documentation website, if we try to duplicate all the docs in the readme it will get very big.

Reply:
The ES6 methods all throw/reject if the global is not available anyway, so I think whatever we do there is additive anyway. I'm thinking about just:
```ts
import { onReady, broadcast } from '@finos/fdc3';
```